### DISCUSSION TITLE
Translations enable

### DISCUSSION THREAD

**Discussion by Komodin-spa** [rustdesk/rustdesk#1791] [General]
Hi! 
 I can translate to Spanish.  I love the Rustdesk, ist a great project. 
If need translate to Spanish, tell me or send me msg.

Thxn!!!
![rustdesk](https://user-images.githubusercontent.com/116502265/197398393-021cee6c-3875-46a9-b4fc-207730abc337.png)

**Comment by rustdesk** [comment]
Thanks, https://github.com/rustdesk/rustdesk/blob/master/src/lang/es.rs

**Comment by Komodin-spa** [comment]
Great! I have corrected some words and meanings. The ideal is to translate it into Iberian Spanish. If you agree, I can improve the translation.

Best regards.
